source/containers.tex

```diff
@@ -11236,7 +11236,10 @@
 requirements\iref{random.access.iterators},
 and
 meets the requirements for
-constexpr iterators\iref{iterator.requirements.general}.
+constexpr iterators\iref{iterator.requirements.general},
+whose \tcode{value_type} is the template parameter \tcode{ElementType}.
```

Comment:
This should be `remove_cv_t<ElementType>`; value types are not cv-qualified.

Reply:
Thank you.

Reply:
Feel free to resolve these comments, @johelegp. I can't since I'm a happily powerless contributor here =)